rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -155,7 +156,7 @@ def build(self):
             exit(0)
         return self
 
-    def run(self):
+    def rocalRun(self):
```

Comment:
why is it renamed to rocalRun()? We have many documentations etc which explains the api as run(). Please revert